src/www/interfaces.php

```diff
@@ -908,8 +908,8 @@ interface_sync_wireless_clones($a_interfaces[$if], false);
             $input_errors[] = gettext("A valid MAC address must be specified.");
         }
         if (!empty($pconfig['mtu'])) {
-            if ($pconfig['mtu'] < 576 || $pconfig['mtu'] > 9000) {
-                $input_errors[] = gettext("The MTU must be greater than 576 bytes and less than 9000.");
+            if ($pconfig['mtu'] < 576 || $pconfig['mtu'] > 9214) {
+                $input_errors[] = gettext("The MTU must be greater than 576 bytes and less than 9214.");
```

Comment:
for not running the risk of bouncing translation strings all the time on updates it should use sprintf %s for both the min and the max value